### ISSUE TITLE
Amazon.Lambda.TestTool for Custom .NET Core Runtimes

### ISSUE THREAD

**Issue by tonysneed** [aws/aws-lambda-dotnet#427]
Are there plans for a version of the Amazon.Lambda.TestTool that will be compatible with a custom runtime for versions of .NET Core (2.2, etc)?

**Comment by normj** [comment]
I would like to, it's just a matter of finding time and demand for it. So if this is a feature people want I encourage people to +1 the issue.

**Comment by tonysneed** [comment]
I'll spread the word!

I can also help with the effort if we discuss the strategy for how this could be implemented.

**Comment by gdevolder** [comment]
Any update on this? Would really appreciate a version upgrade!!

**Comment by miladsleiman** [comment]
yeah, it would be great if this can work in 2.2 as it's not allowing me to use some libraries I have that are 2.2 only.

**Comment by CDargis** [comment]
+1

**Comment by DieterDR** [comment]
+1

**Comment by shivawahi** [comment]
+1

**Comment by rv-amadden** [comment]
+1

**Comment by normj** [comment]
We are working on a new version of the test tool that does support this scenario. We just released the first early preview and admittedly our documentation is light and focused on Aspire integration. In that case the test tool runs out of process and Visual Studio is debugging your Lambda project. In this case the test tool is agnostic to what version of .NET you are using which should cover this use case.

Closing this as it will be handled with the new tooling. Checkout this tracker for the progress of the new tooling. https://github.com/aws/integrations-on-dotnet-aspire-for-aws/issues/17
